### PR TITLE
Add inventory source detail sync button

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
Addresses issue comment: https://github.com/ansible/awx/issues/6574#issuecomment-629341841

* Update source label to display full source name
* Add option tooltips
* Add sync button

Screen Shot 2020-05-19 at 1 40 10 PM

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - UI

### CODE REVIEW

**Review by dsesami — APPROVED** [review]
Looks really great from a QE standpoint. 👍

File: awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx [hunk 1]
```diff
@@ -53,12 +63,28 @@ function InventorySourceDetail({ inventorySource, i18n }) {
...
+    result: sourceChoices,
+    error,
+    isLoading,
+    request: fetchSourceChoices,
+  } = useRequest(
+    useCallback(async () => {
+      const { data } = await InventorySourcesAPI.readOptions();
+      return Object.assign(
+        ...data.actions.GET.source.choices.map(([key, val]) => ({ [key]: val }))
+      );
```
Comment (keithjgrant) [inline awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx:71]: nice. I learned a new trick today :)

File: awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
   UserDateDetail,
 } from '../../../components/DetailList';
 import ErrorDetail from '../../../components/ErrorDetail';
+import useRequest from '../../../util/useRequest';
 import { InventorySourcesAPI } from '../../../api';
 
+const QuestionCircleIcon = styled(PFQuestionCircleIcon)`
+  margin-left: 10px;
+`;
+
```
Comment (keithjgrant) [inline awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx]: We have a `<FieldTooltip>` component that takes care of this and some mouseover behavior we want to keep consistent between tooltips. (components/FormField/FieldTooltip)

**Review by keithjgrant — APPROVED** [review]
Looks good. Just one recommendation for the tooltips

File: awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx (hunk 2 above)
Comment (marshmalien) [reply awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx]: good catch

**Pull request merged**